test/functional/feature_segwit.py

```diff
@@ -117,8 +117,12 @@ def run_test(self):
 
         balance_presetup = self.nodes[0].getbalance()
         self.pubkey = []
-        p2sh_ids = []  # p2sh_ids[NODE][VER] is an array of txids that spend to a witness version VER pkscript to an address for NODE embedded in p2sh
-        wit_ids = []  # wit_ids[NODE][VER] is an array of txids that spend to a witness version VER pkscript to an address for NODE via bare witness
+        # p2sh_ids[NODE][VER=0] is an array of txids that spend to P2WPKH scripts to an address for NODE embedded in p2sh
```

Comment:
maybe instead of `VER` you can say `TYPE` since 0 -> P2WPKH and 1 -> P2WSH in both cases?

Reply:
This is very true haha, changed!